tests/s3/versioning/test_versioning_multipart.py

```diff
@@ -0,0 +1,196 @@
+        self.log.info("STARTED: Test multipart upload in a versioning enabled bucket.")
+        self.log.info("Step 2: PUT Bucket versioning with status as Enabled")
+        res = self.s3ver_test_obj.put_bucket_versioning(bucket_name=self.bucket_name)
+        assert_utils.assert_true(res[0], res[1])
+        self.log.info("Step 3-7: Upload multipart object to a bucket")
+        versions = {}
+        s3ver_cmn_lib.upload_version(self.s3mp_test_obj, self.bucket_name,
+                                                self.object_name, self.test_file_path,
+                                                versions_dict=versions, is_multipart=True,
+                                                total_parts=2, file_size=10, is_unversioned=False)
+        self.log.info("Step 8: List Object Versions")
+        s3ver_cmn_lib.check_list_object_versions(self.s3ver_test_obj, bucket_name=self.bucket_name,
+                                                 expected_versions=versions)
+        self.log.info("Step 9: Check GET/HEAD Object")
+        s3ver_cmn_lib.check_get_head_object_version(self.s3test_obj, self.s3ver_test_obj,
+                                                    bucket_name=self.bucket_name,
+                                                    object_name=self.object_name)
+        self.log.info("Step 10: Check GET/HEAD Object with versionId")
+        s3ver_cmn_lib.check_get_head_object_version(self.s3test_obj, self.s3ver_test_obj,
+                                                    bucket_name=self.bucket_name,
+                                                    object_name=self.object_name,
+                                                    version_id=versions[self.object_name]
+                                                    ["version_history"][0])
+        self.log.info("ENDED: Test multipart upload in a versioning enabled bucket.")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.tags("TEST-41286")
+    def test_mpu_versioning_suspended_bkt_41286(self):
+        """ Test multipart upload in a versioning suspended bucket """
+        self.log.info("STARTED: Test multipart upload in a versioning suspended bucket.")
+        self.log.info("Step 2: PUT Bucket versioning with status as Suspended")
+        res = self.s3ver_test_obj.put_bucket_versioning(bucket_name=self.bucket_name,
+                                                         status="Suspended")
+        assert_utils.assert_true(res[0], res[1])
+        self.log.info("Step 3-7: Upload multipart object to a bucket")
+        versions = {}
+        s3ver_cmn_lib.upload_version(self.s3mp_test_obj, self.bucket_name,
+                                                self.object_name, self.test_file_path,
+                                                versions_dict=versions, is_multipart=True,
+                                                total_parts=2, file_size=10, is_unversioned=False,
+                                                chk_null_version=True)
```

Comment:
is_unversioned does not need to be passed here

Reply:
yup